<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="AddSequences">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Sequence</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sequence_">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="sequences">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>901</width>
     <height>651</height>
    </rect>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FCFS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>FCFS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="StartTime">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>90</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="CylinderMax">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>start </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CYLINDER_MAX </string>
   </property>
  </widget>
  <widget class="QPushButton" name="SSTF">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>SSTF</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Combo_direction">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Right</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Left</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Direction</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SCAN">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>SCAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CSCAN">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>CSCAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CLOOK">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>CLOOK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LOOK">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>LOOK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Headmovement">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>640</y>
     <width>113</width>
     <height>51</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Headmovement</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
